mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -242,7 +299,7 @@ DistanceSensorItem::Ptr DistanceSensorItem::create_item(DistanceSensorPlugin *ow
 		}
 
 		// optional
-		pnh.param("covariance", p->covariance, 0);
+		if (pnh.getParam("covariance", p->covariance) >= 0) p->cov_is_def = true;
```

Comment:
What do you mean? What do I put in https://github.com/mavlink/mavros/pull/296/files#diff-2adf7423447a0e5f4880b56161efdda9R236 then?